lm_eval/api/model.py

```diff
@@ -282,6 +282,11 @@ class TemplateLM(LM):
     def eot_token_id(self):
         pass
 
+    @property
+    @abc.abstractmethod
+    def bos_or_eos_token_id(self):
+        pass
```

Comment:
comment `# it used as prefix for rolling loglikelihood` will be also good one here, for transparency

Reply:
and then, this will be probably better to rename like `rolling_prefix_token_id`, because next time this will be some other token, mask for example.

Reply:
I am good with `bos_or_eos_token_id` for a descriptive name of what this is!